tofhir-server/src/main/scala/io/tofhir/server/endpoint/JobEndpoint.scala

```diff
@@ -42,7 +42,25 @@ class JobEndpoint(jobRepository: IJobRepository, mappingRepository: IMappingRepo
           }
         } ~ pathPrefix(SEGMENT_TEST) { // test a mapping with mapping job configurations, jobs/<jobId>/test
           pathEndOrSingleSlash {
-            testMappingWithJob(projectId, jobId)
+            withRequestTimeoutResponse { _ =>
+              addCORSHeaders(
+                HttpResponse(
+                  status = StatusCodes.RequestTimeout,
+                  entity = HttpEntity.Strict(
+                    ContentTypes.`application/json`,
+                    ByteString(
+                      """Status Code: 408
+                        |Type: https://tofhir.io/errors/RequestTimeout
+                        |Title: Request Timeout
+                        |Detail: There are too many rows in the data source CSV.
```

Comment:
Let 's update this message: The server could not complete the test because of the large data. Try increasing the timeout config and test again.